### PR TITLE
Remove PanicOnError and use globalLogger

### PR DESCRIPTION
**Pull request by xino12**
#### Description of the changes

Changing panicOnError for logOnError so we avoid people using panic and instead using log package.

### Author

- [x] add a risk label after carefully considering the "blast radius" of your changes
- [x] describe the _intent_ of your changes in the description. don't just rewrite your code in prose
- [x] assign at least one reviewer

### Reviewer

- [ ] review code for readability
- [ ] verify that high risk behavior changes are well tested
- [ ] check license for any new external dependency
- [ ] ask questions about anything that isn't clear and obvious
- [ ] approve the PR when you consider it's good to merge

### CODE REVIEW

File: scaffold/resource/tmpl/src/integration_remote.go.tmpl [hunk 1]
```diff
@@ -23,51 +24,52 @@ var (
...
-	panicOnErr(i.Publish())
+	logOnError(i.Publish())
 }
 
-func panicOnErr(err error) {
+func logOnError(err error) {
 	if err != nil {
-		panic(err)
+		log := log.NewStdErr(true)
+		log.Errorf("%s", err)
```
Comment (xino12) [inline scaffold/resource/tmpl/src/integration_remote.go.tmpl]: Maybe use globalLogger instead?

Comment (varas) [reply scaffold/resource/tmpl/src/integration_remote.go.tmpl]: indeed!, remove this function

Comment (varas) [reply scaffold/resource/tmpl/src/integration_remote.go.tmpl]: we decided using global logger is ok, so let's go with that

Comment (varas) [reply scaffold/resource/tmpl/src/integration_remote.go.tmpl]: btw creating a new logger instance each time is not ok

**Pull request merged**